RHDEVDOCS-6054: Content creation for GitOps 1.10.6 RN (#75961)
openshift/openshift-docs: modules/gitops-release-notes-1-10-6.adoc

// Module included in the following assembly:
//
// * release_notes/gitops-release-notes.adoc

:_mod-docs-content-type: REFERENCE

[id="release-notes-for-gitops-1-10-6_{context}"]
= Release Notes for {gitops-title} 1.10.6

{gitops-title} 1.10.6 is now available on {OCP} 4.12, 4.13, and 4.14.

[id="errata-updates-1-10-6_{context}"]
== Errata updates

[id="rhsa-XXXX:NNNN-gitops-1-10-6-security-update-advisory_{context}"]
=== RHSA-2024:131905-03 - {gitops-title} 1.10.6 security update advisory

> Issued: 2024-05-15

Reviewer: Please change the date to 23rd

Author: Fixed.

> Issued: 2024-05-23

Reviewer: ```suggestion
Issued: 2024-05-28
```

Author: Fixed.

The list of security fixes that are included in this release is documented in the following advisory:

> * link:https://access.redhat.com/errata/RHSA-2024:3369-03[RHSA-2024:3369-03]

Reviewer: I am not able to access the link

```suggestion
* link:https://access.redhat.com/errata/RHSA-2024:3369[RHSA-2024:3369-03]
```

Author: Fixed.

If you have installed the {gitops-title} Operator, run the following command to view the container images in this release:

[source,terminal]
----
$ oc describe deployment gitops-operator-controller-manager -n openshift-operators
----

[id="fixed-issues-1-10-6_{context}"]
== Fixed issues

> * Before this update, pods in a different namespace could access the Redis server on port 6379 to obtain read and write access to the data. This issue has been fixed in this release by enabling secure authentication.

Reviewer 2: ```suggestion
* Before this update, pods in a different namespace could access the Redis server on port `6379` to obtain read and write access to the data. This update fixes the issue by enabling secure authentication.
```

Author: Fixed.
